boards/arm/nucleo_l433rc/pinmux.c

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2019 Libre Solar Technologies GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <init.h>
+#include <drivers/pinmux.h>
+#include <sys/sys_io.h>
+
+#include <pinmux/stm32/pinmux_stm32.h>
+#if defined(CONFIG_GPIO)
```

Comment:
This should depend on `CONFIG_PINMUX`. I believe that a board without this setting is pretty much useless anyway, so it's probably a redundant check...

Reply:
I added this because I was building the bootloader for this board, with no GPIO support -- for minimum size, and this was raising an issue. I don't believe `CONFIG_PINMUX` depends on `CONFIG_GPIO`. I can add both.

Reply:
Actually the file is not even compiled if `CONFIG_PINMUX=n`, so I think it's ok then.